dataland-frontend/src/components/resources/frameworkDataSearch/lksg/LksgCompanyDataTable.vue

```diff
@@ -111,7 +127,7 @@ export default defineComponent({
     },
   },
   mounted() {
-    this.kpiDataObjectsToDisplay = Array.from(this.kpiDataObjects.values());
+    this.kpiDataObjectsToDisplay = this.arrayOfKpiDataObjects;
```

Comment:
`kpiDataObjectsToDisplay` is pointless.  It just bears the value of the prop `arrayOfKpiDataObjects`, and also causes a type violation here, because 
`kpiDataObjectsToDisplay` is of type `Array<KpiDataObject>`
and
`arrayOfKpiDataObjects` is of type `Array<KpiDataObject> | undefined`

So why not delete `kpiDataObjectsToDisplay`?

Reply:
done

Reply:
Thanks. I just noticed that the prop `tableDataTitle` is never used. Could you delete it?

Same goes for that prop with the same name in the file `CompanyDataTable.vue`

Reply:
done, waiting for green CI

Reply:
It is still there as set prop in some other components.  Search fo `tableDataTitle` in the `src` folder of `dataland-frontend` to see it.

Reply:
The CI is failing because of this statement in the component-test of a component that used to have `tableDataTitle`

`expect(wrapper.vm.tableDataTitle).to.be.a("string").that.is.empty`

Reply:
Fixed. Closing it.